Library/Homebrew/utils/portable-ruby-sha256/arm64-darwin

```diff
@@ -0,0 +1,2 @@
+ruby_TAG=arm64_big_sur
```

Comment:
`utils` feels a bit weird here, how about `Library/Homebrew/vendor/` somewhere e.g. `Library/Homebrew/vendor/portable-ruby-arm64-darwin`?

Reply:
Moved to the suggested location (original location was taken from the proposal in the issue).